Sources/Extensions/AppIntents/Widget/Details/WidgetDetailsAppIntent.swift

```diff
@@ -0,0 +1,51 @@
+import AppIntents
+import AudioToolbox
+import Foundation
+import Shared
+
+@available(iOS 17.0, macOS 14.0, watchOS 10.0, *)
+struct WidgetDetailsAppIntent: WidgetConfigurationIntent {
+    static let title: LocalizedStringResource = "Details"
+    static let description = IntentDescription("Display states using from Home Assistant in text")
+    
+    @Parameter(title: "Server", default: nil)
```

Comment:
Please localize all parameter titles in this file

Reply:
I can’t figure out localized string resources, I’ll try again tomorrow.

Reply:
After adding strings to Localizable.strings and building the app, swiftgen will create a wrapper around it that you can access by using `L10n.something.something`

Reply:
Yeah, I saw that elsewhere in the app, but the app intent parameters seem to want LocalizedStringResource instances. I can only seem to convert a string literal to one, not a string. They seem to just get the string from the strings files themselves using the key, but I can't get them to work, so I might just be missing something.

Reply:
Hmm interesting, I'll take a look tomorrow and get back to you

Reply:
Figured it out. The strings file needed to be in the app target for it to find the keys.

Reply:
If this is the only issue, `LocalizedStringResource` has an option to specify the bundle, which would avoid the need to making Localizable.string available to the main target directly.

But while testing I couldn't make it work either way